app/Environment.php

```diff
@@ -53,7 +53,7 @@ private static function setConfig()
     public static function env()
```

Comment:
Could you please not commit changes to this file

Reply:
sure